src/ResourceManager/DevSpaces/Commands.DevSpaces/Utils/ConversionUtils.cs

```diff
@@ -0,0 +1,45 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+using System;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.DevSpaces.Utils
+{
+    public static class ConversionUtils
```

Comment:
Shouldn't be necessary after converting to ResourceIdentifier.

Reply:
ConversionUtils.TryParseResourceId is implemented on top of ResourceIdentifier to make sure that we are parsing DevSpace or AKS ID.